
nav_msgs/Odometry gps_global
---
nav_msgs/Odometry gps_local

